<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Goods_Widget</class>
 <widget class="QDialog" name="Goods_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>359</width>
    <height>182</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Goods_Widget{
	border: 5px solid rgb(255, 110, 255);
	border-color: rgb(255, 123, 169);
	border-radius: 12px;
	background-color: rgb(255, 208, 245);
}

QPushButton{
	background-color: rgb(255, 229, 235);
	border: 3px solid rgb(255, 125, 153);
    border-radius: 12px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
	border: 3px solid rgb(255, 174, 192);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,5">
   <item>
    <widget class="QLabel" name="label_pic">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_store">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>库存</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_price">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>单价</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
